spring-core/src/main/java/com/baeldung/resource/LoadResourceConfig.java

```diff
@@ -0,0 +1,55 @@
+package com.baeldung.resource;
+
+import com.google.common.io.CharStreams;
+import org.apache.commons.io.FileUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.Resource;
+import org.springframework.util.FileCopyUtils;
+import org.springframework.util.StreamUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+
+@Configuration
+public class LoadResourceConfig {
+    @Value("classpath:resource.txt")
+    private Resource resource;
+
+    @Value("#{T(org.apache.commons.io.FileUtils).readFileToString(" + "T(org.springframework.util.ResourceUtils).getFile('classpath:resource.txt')" + ")}")
```

Comment:
I'm not sure what the aim is here. If it's to show how much you can put into a SPEL statement, then great. If it's more to show that you can wire in via SPEL, then perhaps we could have a static function somewhere with the _meat_ of this code in, and you can just wire in calling that static function.

This SPEL code is broken up with `" + "` but not line breaks, making it harder to read. It could be broken with `" + <linebreak>` before each `.functionCall(`